canvas_modules/common-canvas/src/common-canvas/svg-canvas-d3.js

```diff
@@ -52,6 +52,10 @@ export default class SVGCanvasD3 {
 		document.addEventListener("mousemove", this.onMouseUpdate.bind(this), true);
 	}
 
+	close() {
+		document.removeEventListener();
```

Comment:
This won't work. In my previous comment I had written in "figure out what params are needed" - however, when I wrote it I surrounded it with angle brackets and unfortunately Gihub doesn't show text in between angle brackets so it wasn't visible to you.   Anyway, you need some parameters in here.